advcpmv-0.9-9.0.patch

```diff
@@ -187,40 +216,54 @@ diff -aur coreutils-9.0/src/copy.c coreutils-9.0-patched/src/copy.c
    /* Ensure a trailing hole is created, so that subsequent
       calls of sparse_copy() start at the correct offset.  */
    if (make_hole && ! create_hole (dest_fd, dst_name, punch_holes, psize))
-@@ -516,9 +668,11 @@
- static bool
+@@ -517,8 +700,16 @@
  lseek_copy (int src_fd, int dest_fd, char *buf, size_t buf_size,
              size_t hole_size, off_t ext_start, off_t src_total_size,
--            enum Sparse_type sparse_mode,
-+            enum Sparse_type sparse_mode, bool move_mode,
+             enum Sparse_type sparse_mode,
++            /* BEGIN progress mod */
++            bool move_mode,
++            /* END progress mod */
              bool allow_reflink,
 -            char const *src_name, char const *dst_name)
-+            char const *src_name, char const *dst_name,
-+            int iCountDown, char ** cProgressField, struct timeval last_time,
-+            int last_size, int iBarLength, struct stat src_open_sb)
++            char const *src_name, char const *dst_name
++            /* BEGIN progress mod */
++            , int iCountDown, char ** cProgressField, struct timeval last_time,
++            int last_size, int iBarLength, struct stat src_open_sb
++            /* END progress mod */
++            )
  {
    off_t last_ext_start = 0;
    off_t last_ext_len = 0;
-@@ -590,10 +744,16 @@
+@@ -590,10 +781,26 @@
           is conservative and may miss some holes.  */
        off_t n_read;
        bool read_hole;
 +
++      /* BEGIN progress mod */
 +      struct timeval a;
 +      struct stat b;
 +
 +      struct progress_status s_progress={iCountDown, cProgressField, last_time, last_size, iBarLength, src_open_sb};
++      /* END progress mod */
 +
        if ( ! sparse_copy (src_fd, dest_fd, buf, buf_size,
                            sparse_mode == SPARSE_NEVER ? 0 : hole_size,
 -                          true, allow_reflink, src_name, dst_name,
 -                          ext_len, &n_read, &read_hole))
-+                          true, allow_reflink, move_mode, src_name,
-+                          dst_name, ext_len, &n_read, &read_hole, &s_progress))
++                          true, allow_reflink,
```

Comment:
Can you please remove these cosmetic one variable per line changes? Makes it very difficult to review.

Also please remove the inline comments /* BEGIN progress mod */ and /* END progress mod */ everywhere. It's evident from the diff and the flow.

Reply:
Remove the comments everywhere in the patch or only those I added?

Reply:
Those you added, to make review easier.

Reply:
Your requested changes have been added. I hope I got all of them.